### ISSUE TITLE
error with optional chaining

### ISSUE THREAD

**Issue by sirenkovladd** [putoutjs/minify#19]
- **Version** (`npm -v`): 10.5.2
- **Node Version** `node -v`: v20.13.1
- **OS** (`uname -a` on Linux): Darwin arm64

error with optional chaining

```js
import { minify } from "@putout/minify";

const source = `Transition && (Transition.running = !1);`;

const Transition = { running: 1 }
eval(source);
const result = minify(source);
console.log(result);
eval(result);
```

```sh
❯ node i.js 
Transition?.running=!1;
undefined:1
Transition?.running=!1;
^^^^^^^^^^^^^^^^^^^

SyntaxError: Invalid left-hand side in assignment
```

**Comment by coderaiser** [comment]
https://github.com/tc39/proposal-optional-chaining-assignment

**Comment by coderaiser** [comment]
Landed in [v3.19.4](https://github.com/putoutjs/minify/releases/tag/v3.19.4) 🎉

**Comment by sirenkovladd** [comment]
thank you
